### PR TITLE
Fix some image/video scroll jumps

### PR DESCRIPTION
**Pull request by robintown**
Images without a blurhash and all videos were not reserving the correct amount of space in the timeline while loading. Fixing this also involved fixing an unreported regression in the blurhash reveal animation, which previously faded to white instead of fading to the actual image.

Type: defect

Related: https://github.com/vector-im/element-web/issues/6236.
Related: https://github.com/vector-im/element-web/issues/20154.

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix some image/video scroll jumps ([\#8182](https://github.com/matrix-org/matrix-react-sdk/pull/8182)).

Preview: https://pr8182--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/messages/MVideoBody.tsx [hunk 1]
```diff
@@ -228,6 +228,18 @@ export default class MVideoBody extends React.PureComponent<IBodyProps, IState>
...
+        if (content.info?.w && content.info?.h) {
+            aspectRatio = content.info.w / content.info.h;
+        }
+        const { w: maxWidth, h: maxHeight } = suggestedVideoSize(
+            SettingsStore.getValue("Images.size") as ImageSize,
+            { w: content.info?.w, h: content.info?.h },
+        );
+
+        // HACK: This div fills out space while the video loads, to prevent scroll jumps
+        const spaceFiller = <div style={{ width: maxWidth, height: maxHeight }} />;
```
Comment (t3chguy) [inline src/components/views/messages/MVideoBody.tsx:241]: What hides this? it seems to always get rendered with both the spinner and video tags unlike in MImageBody?

Does it just get pushed off by the video/spinner and cut by the overflow none? Can we do a similar state.loaded solution to not pollute the DOM too much?

**Review by t3chguy — APPROVED** [review]
looks sane otherwise

File: src/components/views/messages/MVideoBody.tsx (hunk 1 above)
Comment (robintown) [reply src/components/views/messages/MVideoBody.tsx:241]: Yep, it currently gets cut off by overflow, will look into hiding it on load

Comment (robintown) [reply src/components/views/messages/MVideoBody.tsx:241]: I tried this, but couldn't get any of `onLoad`, `onLoadedMetadata`, `onLoadedData` to fire, so let's live with this bit of DOM pollution for now.

**Pull request merged**